dotnet/src/SemanticKernel.Abstractions/Data/RecordOptions/VectorSearchOptions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.SemanticKernel.Data;
+
+/// <summary>
+/// Options for vector search.
+/// </summary>
+[Experimental("SKEXP0001")]
+public class VectorSearchOptions
+{
+    /// <summary>
+    /// Gets the default options for vector search.
+    /// </summary>
+    public static VectorSearchOptions Default { get; } = new VectorSearchOptions();
```

Comment:
`VectorSearchOptions` parameter is optional in interface, and default is `null`, so we will initialize it on our side by using empty constructor in all cases. So, I'm wondering if this public property is required at all?

Reply:
I'm using it in the implementations as the default when none is provided.  This has two implications:

1. Someone can set default settings here that apply globally if no options are provided
2. I don't have to construct a new instance of options each time to get a default, meaning fewer allocations.

1 probably isn't super useful, considering the nature of the settings on here, while 2 is somewhat useful.